### ISSUE TITLE
Insufficient Notification of Project Saving in Admin Interface

### ISSUE THREAD

**Issue by Santoz2** [ManifoldScholar/manifold#2559]
## Expected Behavior
Whenever a user makes an action, the results of that action should be clear. In this case, there should be clear notification about whether or not the project has completed being saved. This applies to both the visual notification and the audio notification.

## Current Behavior
There is currently a visual indicator of the progress of saving a project. On the top of the browser window, there is a green progress bar that proceeds from the left of the screen to the right, then disappears. This is the only indicator that the project has been saved; some users may not understand what the green progress bar means, and Screen Readers do not announce it at all. 

## Steps to Reproduce
1. Navigate to the Admin Interface (via Enter Admin Mode)
2. Navigate to an editable project, landing on its General Page
3. Navigate to the Save Project button on the bottom of the page
4. Activate the Save Project button and notice that the only indication that the project has been saved is the disappearance of the progress bar on the top. 

## Testing Environment
NVDA 2019.2.1
ESR Firefox 68.4.1 esr

**Comment by zdavis** [comment]
@bot bug

**Comment by zdavis** [comment]
Thank you for taking the time to report this bug. We've reviewed the issue and agree that this should be fixed. We’re adding a “planned” label to indicate that we consider this bug as part of our current work plan.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_

**Comment by Santoz2** [comment]
Accessibility Triage Notes: UW's ATX team considers this to be a priority 2 issue, as the user's task is completed, and saving can be confirmed, after some effort.
